src/index.html

```diff
@@ -5,9 +5,60 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/stars.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+      <div class="card__image"></div>
+
+      <p class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__stars-and-reviews">
+        <section class="card__stars card__stars--4">
+          <div class="card__stars card__stars--star"></div>
+          <div class="card__stars card__stars--star"></div>
+          <div class="card__stars card__stars--star"></div>
+          <div class="card__stars card__stars--star"></div>
+          <div class="card__stars card__stars--star"></div>
+        </section>
```

Comment:
```suggestion
      <div class="card__stars-and-reviews">
        <div class="card_stars stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```

Reply:
let's make stars the separate block, cause we should have the opportunity to reuse stars

Reply:
[DEMO LINK](https://in7264.github.io/layout_product-cards/)
[TEST REPORT LINK](https://in7264.github.io/layout_product-cards/report/html_report/)